File: sistemastreamingjs/suscripcion.js
class Suscripcion {
    constructor(nombre = "Básica", precio = 0.00) {
        this.nombre = nombre;
        this.precio = precio;
    }

    toString() {
        return `Nombre: ${this.nombre}, Precio: $${this.precio}`;
    }
}

module.exports = Suscripcion;

File: sistemastreamingruby/suscripcion.rb
class Suscripcion
    attr_accessor :nombre, :precio
  
    def initialize(nombre = "Básica", precio = 0.00)
      @nombre = nombre
      @precio = precio
    end
  
    def to_s
      "Nombre: #{@nombre}, Precio: $#{@precio}"
    end
  end
  